### DISCUSSION TITLE
Pimcore Academy - Struggling with the document properties question

### DISCUSSION THREAD

**Discussion by michaelopre** [pimcore/pimcore#15267] [Q&A / Support] [answered]
Hi!

I'm struggling with the Question 7 for documents, Junior developer:

Question 7: Which statements concerning properties are correct?

1. Properties can be used to influence navigation building.
2. Custom Properties need to be considered in custom code (business logic or template) to have any effect to the application.
3. Properties can be inherited via content master document.
4. The language of individual documents can be defined with a system property.

From my point of view they are:
1. because it is explained exactly how I can influence the output of the navigation using properties and the Navigation Builder.
2. also here I see in the video and the documentation no indication that this is not so.
3. again, I can set the properties in the master document and set the inherit checkbox to inherit them to the child documents.

Unfortunately this is not correct, but I can't find any hint that I can set the language of individual documents with a system property.

Or does the language/localization under System Settings count as a property?

Can anybody help, because I only have one try left?

Best,
Michael

**Comment by NiklasBr** [comment, marked as answer]
1. ✔️ because e.g. Navigation Name is used to both determine the name and visibility in the navigation.
2. ✔️ the only way to have any programmatic use of a custom property is to write code/template that uses it. Otherwise it is just a manual data store.
3. ⛔ (but I agree that this question was confusing).
4. ✔️ yes you can change the language of an individual content page, which is a property.

![image](https://github.com/pimcore/pimcore/assets/279826/b29e4090-c16f-42b6-a9e7-8fd300a39744)

**Reply by michaelopre** [reply]
Hey Niklas,

thanks a lot lot! You're right! 
It worked! But know I had an mistake in question 8; although I knew it. 
New, new try ;-)

Thank you very much and have a great day!

Best,
Michael
